src/cinnamon-recorder.c

```diff
@@ -1263,10 +1255,7 @@ recorder_open_outfile (CinnamonRecorder *recorder)
                   {
                     /* Appends date as YYYYMMDD */
                     GDate date;
-                    GTimeVal now;
-                    g_get_current_time (&now);
                     g_date_clear (&date, 1);
-                    g_date_set_time_val (&date, &now);
```

Comment:
I'm not sure what the point of this is. This is just for YYYY-MM-DD and i don't know what time has to do with anything, it doesn't seem to be used in the final output

Reply:
used here: https://github.com/linuxmint/Cinnamon/blob/master/js/ui/main.js#L208-L211

Using the screen recorder I get a file named `cinnamon-20210531-1.webm`